### DISCUSSION TITLE
Can I get the plots sum for all of the Harvesters ,In main machine?

### DISCUSSION THREAD

**Discussion by jzmexec** [Chia-Network/chia-blockchain#5938] [Q&A]
Can I get the plots sum for all of the Harvesters ,In main machine, with 'chia farm summary'? Now, I just get the plots in man machine.
Is it wrong? How can I get the sum?

**Comment by rcamarda390** [comment]
I dont think so, not at this time. vids I watched about headless PI with windows UI said there are limitations like this.
